packages/ketcher-core/src/domain/entities/sgroup.ts

```diff
@@ -504,8 +505,18 @@ export class SGroup {
       });
       braketBox = !braketBox ? bbb : Box2Abs.union(braketBox, bbb!);
     });
-    const vext = new Vec2(0.2, 0.4);
-    if (braketBox) braketBox = (braketBox as Box2Abs).extend(vext, vext);
+    if (!render) render = global._ui_editor.render;
```

Comment:
Could you, please, describe the case when render is not available?

Reply:
when copy or cut a s-group in on the clipboard